README.md

```diff
@@ -1,2 +1,2 @@
 # mendelian-search-application
-The Mendelian Search Application is a PhenoTips application which will provide users with a method of querying for a candidate gene and a set of phenotypes.
+The Mendelian Search Application is a PhenoTips application which will provide users with a method of querying for patients with matching variants in a user-specified gene and a matching user-specified phenotype. 
```

Comment:
include a blurb about how to build and deploy this, as well as it's dependencies.